### PR TITLE
Calculator section

### PR DESCRIPTION
**Pull request by IvanSidorenkov**

### CODE REVIEW

File: src/sections/calculator/Calculator.module.scss [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+  max-width: 1100px;
+  width: 100%;
+  margin: 80px auto 0;
+  background-color: $lightGrey;
+  padding: 45px 0 30px 46px;
+  display: flex;
+  height: 307px;
+
+  &:before {
+    content: url("../../public/assets/images/png/tree.png");
```
Comment (AndreiLavrov) [inline src/sections/calculator/Calculator.module.scss:11]: maybe use |'| not |"| ?

File: src/sections/calculator/Calculator.module.scss [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+    font-size: 22px;
+    line-height: 27px;
+    text-align: right;
+    color: $black;
+    -moz-appearance: textfield;
+    padding-right: 16px;
+
+    &:focus {
+      outline: none;
+      background: #F5F5F5;
```
Comment (AndreiLavrov) [inline src/sections/calculator/Calculator.module.scss:106]: vars?

File: src/sections/calculator/Calculator.module.scss [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+      border: 1px solid #9E9E9E;
+      color: #AFAFAF;
+      caret-color: #1673FF;
+    }
+
+    &::-webkit-inner-spin-button {
```
Comment (AndreiLavrov) [inline src/sections/calculator/Calculator.module.scss:112]: why so specific?

File: src/sections/calculator/Calculator.tsx [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+        </div>
+        <div className={styles.divider} />
+        <div className={styles.textGroup}>
+          <p className={`p1 ${styles.descriptionTextTotal}`}>конечная стоимость покупки, pln</p>
+          <p className={`p1 ${styles.descriptionTotal}`}>{totalResult}</p>
+        </div>
+      </section>
+      <section className={styles.vatWrapper}>
+        <p>VAT:</p>
+        {vatPercentage.map((vat) => <VatElement
```
Comment (AndreiLavrov) [inline src/sections/calculator/Calculator.tsx:108]: strange syntax.
I usually use in such way
{vatPercentage.map(vat => (
        <VatElement
          key={vat}
          text={vat}
          activeVat={activeVat}
          handleActiveVat={handleActiveVat}
        />
      ))}

**Review by AndreiLavrov — APPROVED** [review]

File: src/sections/calculator/Calculator.module.scss (hunk 3 above)
Comment (IvanSidorenkov) [reply src/sections/calculator/Calculator.module.scss:112]: For removing arrows on <input type="number" />

File: src/public/assets/images/png/clipPath.png
Comment (ssavitski) [inline src/public/assets/images/png/clipPath.png:1]: There is no sense to keep whole line, pattern is repeated. There is enough to store just a fragment 
image

and repeat it vertically

File: src/public/assets/images/png/tree.png
Comment (ssavitski) [inline src/public/assets/images/png/tree.png:1]: The same as comment above

File: src/sections/calculator/Calculator.module.scss [hunk 5]
```diff
@@ -0,0 +1,368 @@
...
+  background-color: $lightGrey;
+  padding: 45px 0 30px 46px;
+  display: flex;
+  height: 307px;
+}
+
+.leftBorderImage {
+  position: absolute;
+  margin-top: -46px;
+  margin-left: -59px;
```
Comment (ssavitski) [inline src/sections/calculator/Calculator.module.scss:15]: We left comments about such numbers

File: src/sections/calculator/Calculator.module.scss [hunk 6]
```diff
@@ -0,0 +1,368 @@
+.calculatorWrapper {
+  position: relative;
+  max-width: 1100px;
+  width: 100%;
+  margin: 80px auto 0;
+  background-color: $lightGrey;
+  padding: 45px 0 30px 46px;
+  display: flex;
+  height: 307px;
```
Comment (ssavitski) [inline src/sections/calculator/Calculator.module.scss:9]: We left comments about such numbers

File: src/sections/calculator/Calculator.module.scss [hunk 7]
```diff
@@ -0,0 +1,368 @@
...
+.leftBorderImage {
+  position: absolute;
+  margin-top: -46px;
+  margin-left: -59px;
+}
+
+.rightBorderImage {
+  position: absolute;
+  margin-top: -46px;
+  margin-right: -13px;
```
Comment (ssavitski) [inline src/sections/calculator/Calculator.module.scss:21]: The same as comment above

File: src/sections/shops/Shops.tsx [hunk 8]
```diff
@@ -11,7 +11,7 @@ const Shops = () => {
       <h2 className={styles.title}>Доставляем ото всюду</h2>
       <div className={styles.shopsWrapper}>
         {SHOPS.map((shop) =>
```
Comment (ssavitski) [inline src/sections/shops/Shops.tsx:13]: you can destructure shop

**Review by ssavitski — APPROVED** [review]

**Pull request merged**